docs/CONTRIBUTING.md

```diff
@@ -62,23 +62,17 @@ The pull request points to that branch, not to specific commits.
 The code style expected for code change contributions is explained in the [style guide][].
 
 The exercise code is checked during [CI][Continuous Integration] for conformance to the spacing and indentation style rules.
-The check uses the GNU `indent` CLI tool.
+The check uses the [`clang-format`][clang-format] CLI tool.
 When adding or making a change to an exercise, you can check that your change conforms to these rules by running the tool locally before committing.
 
-On Linux `indent` should be available for install via your distribution's package manager.
-On a Mac you can install the package `gnu-indent` using [Homebrew][].
+On Linux `clang-format` should be available for install via your distribution's package manager.
+On a Mac you can install the package `clang-format` using [Homebrew][].
 
-Indent can be run on all files by executing `indent.sh`.
-To manually run it on a single file, you can execute:
+clang-format can be run on all files by executing `format.sh`.
+To manually run it on a single file, you can execute the following from the same location as the `.clang-format` file:
```

Comment:
`clang-format` actually crawls up the directory tree to find the nearest parent folder containing a `.clang-format`. Just FYI, I don't think we need to make this more confusing by adding that nuance here.